DevVehicle35-Motors/App/Program.cs

```diff
@@ -23,6 +25,9 @@
             case 3:
                 QuadInteraction.BuildQuad();
                 break;
+            case 4:
+                new MiniBus();
+                break;
```

Comment:
Ths is just for a common way of implementing the build of an vehicle, so just as in Line 23 and Line 26 should use an MiniBusInteraction with a BuildMiniBus method to instantiate an Minibus

Reply:
I created a new interaction only to a minibus, ty